Dockerfile

```diff
@@ -32,6 +32,9 @@ LABEL name="Red Hat Messagign QE - Java CLI Image" \
 
 USER root
 
+#Install fallocate util to create files of specific size
```

Comment:
I don't like this comment. People can be assumed to know what fallocate does. We need to say why we need it to have in the image.

> If I grep this entire project, nothing uses no fallocate. So, can I delete it?

And the answer should be an obvious no, thanks to this comment.